src/java.base/share/classes/java/util/concurrent/ForkJoinTask.java

```diff
@@ -1072,7 +1072,7 @@ public final void quietlyInvoke() {
 
     /**
      * Tries to join this task, returning true if it completed
-     * (possibly exceptionally) before the given timeout and
+     * (possibly exceptionally) before the given timeout elapsed and
      * the current thread has not been interrupted.
```

Comment:
```suggestion
     * (possibly exceptionally) before the given timeout elapsed and if
     * the current thread has not been interrupted.
```
Would it be clearer with another _“if”_? I assume, the meaning is “…returning true if it completed … and if the current thread has not been interrupted.”

Reply:
I'd leave it to @DougLea.

Reply:
The extra "if" doesn't seem to add clarity.